webapp/commonComponents/validationTooltip.js

```diff
@@ -1,32 +1,32 @@
 import React from 'react'
 
-import * as ValidationUtils from '@webapp/utils/validationUtils'
+import ValidationFieldMessages from '@webapp/commonComponents/validationFieldMessages'
 import * as Validation from '@core/validation/validation'
 
 import Tooltip from './tooltip'
 
-import { useI18n } from './hooks'
-
 export default ({ validation, className, showKeys, children }) => {
-  const i18n = useI18n()
-
   const isValid = Validation.isValid(validation)
 
-  const type = isValid
-    ? ''
-    : Validation.isWarning(validation)
-      ? 'warning'
-      : 'error'
+  let type = ''
+  if (Validation.isError(validation)) {
+    type = 'error'
+  } else if (Validation.isWarning(validation)) {
+    type = 'warning'
+  }
 
-  const messagesHtml = isValid
+  const content = isValid
     ? null
-    : ValidationUtils.getValidationFieldMessagesHTML(i18n, showKeys)(validation)
+    : React.createElement(ValidationFieldMessages, { validation, showKeys })
+
+  const show = Validation.isWarning(validation) || Validation.isError(validation)
```

Comment:
can it be just !Validation.isValid(validation)?

Reply:
In some cases you can have isValid = false with no tooltip content